Stripe/STPTelemetryClient.m

```diff
@@ -111,8 +100,9 @@ - (NSString *)language {
 - (NSString *)platform {
     struct utsname systemInfo;
     uname(&systemInfo);
-    NSString *deviceType = @(systemInfo.machine);
-    return deviceType ?: @"";
+    NSString *deviceType = @(systemInfo.machine) ?: @"";
```

Comment:
This seems like a strange type coercion? here? `@()` returns NSNumber but assigning to NSString?